autospotting.go

```diff
@@ -124,6 +128,15 @@ func (c *cfgData) parseCommandLineFlags() {
 			"\tset your bid price to be higher than the on demand price to reduce the chances that your\n"+
 			"\tspot instances will be terminated.")
 
+	flag.Float64Var(&c.SpotPriceMultiplier, "spot_price_multiplier", 1.1,
+		"Multiplier for the spot price. Placing bids a fraction of a cent over the current price\n"+
+			"\twould aggressively optimize for cost savings since the last runtime hour before spot termination\n"+
+			"\tis free of charge.")
```

Comment:
It turns out after the per-second billing the last hour is no longer free of charge. 

As of now the main benefit is that it protects the group from running spot instances that got significantly more expensive than when they were initially launched, but still somewhat less than the on-demand price.

For example:
- on-demand price: 1
- spot-price: 0.2 but increased to 0.9 over time
- this instance would be terminated by this new policy as soon as the price was exceeding 0.22 (using the vehiculated 10% price buffer we want to use as default the bid price would be 0.22, instead of 1).

With the default policy the spot instance in the above example may run forever after this significant price increase unless the price goes over 1.

This allows us to find other spot instance types that may be priced better as of the moment when the price increased.

Reply:
I'm going ahead with percentage value as it seems more intuitive. Is that cool?